packages/language-server/src/completion/completions.ts

```diff
@@ -663,7 +663,15 @@ function getDefaultValues(
   currentLine: string,
   lines: string[],
 ): CompletionItem[] {
-  const suggestions: CompletionItem[] = []
+  const suggestions: CompletionItem[] = [
+    {
+      label: 'dbgenerated("")',
+      kind: CompletionItemKind.Function,
+      documentation: 'Use a default value from your database',
```

Comment:
The SQL definition of the default value which is generated by the database. This is not validated by Prisma. 
Example: root Float @default(dbgenerated("sqrt(4)")) yields
Create Table(
     root Float Not Null Default sqrt(4)
)

Reply:
```suggestion
      documentation: 'The SQL definition of the default value which is generated by the database. This is not validated by Prisma.',
```